emr-dynamodb-hadoop/src/main/java/org/apache/hadoop/dynamodb/DynamoDBConstants.java

```diff
@@ -62,6 +62,9 @@
   String MAX_BATCH_SIZE = "dynamodb.max.batch.size";
   String MAX_ITEMS_PER_BATCH = "dynamodb.max.batch.items";
 
+  String DELETION_MODE = "dynamodb.deletion.mode";
```

Comment:
[questions] not a blocker for this feature, may I ask the reason/difference from deleting the table from console or DynamoDB CLI other than via this tools? just want to know the motivation behind this feature?

Reply:
Some context here- https://github.com/awslabs/emr-dynamodb-connector/issues/70

Reply:
thanks @ajay25  !!